app/connectors/CustomsDeclareExportsMovementsConnector.scala

```diff
@@ -62,4 +64,42 @@ class CustomsDeclareExportsMovementsConnector @Inject()(appConfig: AppConfig, ht
 
   private def logFailedExchange(`type`: String, exception: Throwable): Unit =
     logger.warn(`type` + " failed", exception)
+
+  def fetchAllSubmissions(providerId: String)(implicit hc: HeaderCarrier): Future[Seq[SubmissionFrontendModel]] =
+    httpClient
+      .GET[Seq[SubmissionFrontendModel]](s"$CustomsDeclareExportsMovementsUrl$FetchAllSubmissionsEndpoint", providerIdQueryParam(providerId))
+      .andThen {
+        case Success(response)  => logger.debug(s"Submissions fetch response. $response")
```

Comment:
Why not use the log methods the other connector methods use?

The intention was for it to be consistent to Kibana searches are easier